app/models/base.py

```diff
@@ -1,7 +1,23 @@
-from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy import String, DateTime, MetaData
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+import pytz
 
+# 한국 시간대 설정
+kst = pytz.timezone("Asia/Seoul")
 
-# Base 클래스는 SQLAlchemy의 모든 ORM 모델 클래스가 상속받는 기본 클래스입니다.
-# 이 클래스는 데이터베이스 테이블과 매핑되는 ORM 모델을 정의하는 데 사용됩니다.
+
+# 3개 다 필요함 공통된 칼럼을 상속받는 Base
 class Base(DeclarativeBase):
-    pass  # 이 클래스 자체는 비어 있지만, 상속받는 클래스들이 테이블을 정의하게 됩니다.
+
+    id: Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid4
+    )
+
+    created_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=lambda: datetime.now(kst)
```

Comment:
```suggestion
        DateTime(timezone=True), server_default=func.now()
```

default=lambda: datetime.now() 는 아래 링크와 사진 처럼 실제 디비에 추가되는 시간이 아닌 앱이 실행될때 호출되므로 실제 만들어진 시간과 차이가 있을 수 있어서 제가 제안한거로 수정해주시면 좋을 거 같습니다!
https://www.reddit.com/r/flask/comments/ypqk40/default_datetimenow_not_using_current_time/?rdt=43605

![스크린샷 2025-04-04 오전 11 18 24](https://github.com/user-attachments/assets/2be7c57e-4094-4fb0-b2f6-bb63e5fd4852)

Reply:
수정하겠습니다!

Reply:
수정했슴닷